app/scripts/modules/core/src/domain/IPipeline.ts

```diff
@@ -28,6 +28,13 @@ export interface IParameter {
   'default': string;
   hasOptions: boolean;
   options: IParameterOption[];
+  comparator?: IParameterCondition;
```

Comment:
should this be `comparator?: IParameterCondition;`

Reply:
I don't know what you're talking about